src/memcached/cache_impl.go

```diff
@@ -136,7 +136,7 @@ func (this *rateLimitMemcacheImpl) increaseAsync(cacheKeys []limiter.CacheKey, i
 	limits []*config.RateLimit, hitsAddend uint64) {
 	defer this.waitGroup.Done()
 	for i, cacheKey := range cacheKeys {
-		if cacheKey.Key == "" || isOverLimitWithLocalCache[i] {
+		if cacheKey.Key == "" || isOverLimitWithLocalCache[i] || limits[i].Unlimited {
```

Comment:
Is this still needed if we lift this out of the cache layer?

Reply:
Removing